src/support_tests.cc

```diff
@@ -197,7 +210,8 @@ bool test_authenticated_encrypt(bool print_all) {
   }
   if (!authenticated_decrypt("aes-256-gcm", cipher, size_encrypt_out, key,
             decrypted, &size_decrypt_out)) {
-    printf("authenticated decrypt for aes-256-gcm failed\n");
+    printf("%s() error, line: %d, authenticated for aes-256-gcm decrypt failed\n",
```

Comment:
? Hmmm .... seems like there's a reason here why you chose to rearrange the terms here, too.

The older message seems more correct.

Will leave this, and prev one, up to you to resolve as appropriate.

Reply:
done